plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/TestMongoConnectorTest.java

```diff
@@ -816,6 +817,17 @@ public void testNativeQueryInvalidArgument()
                 "Can't parse 'filter' argument as json");
     }
 
+    @Test
+    public void testNativeQueryWithCaseInSensitiveNameMatch()
+    {
+        String tableName = "Test_Case_Insensitive" + randomNameSuffix();
+        MongoCollection<Document> collection = client.getDatabase("tpch").getCollection(tableName);
```

Comment:
Database name should contain uppercase.

Reply:
> Database name should contain uppercase.

I changed it with variable `databaseName` with upper casing and randomSuffix